### PR TITLE
Update config

### PR DESCRIPTION
**Pull request by forabi**

### CODE REVIEW

**Comment by forabi** [comment]
@msafih I think this is ready for merging. We still need to update the other repos to extend the new common config. Should I also remove the `common-pkg` stuff from `package.json`?

**Comment by msafih** [comment]
@forabi Yeah, let's remove `commonpkg` from every where. I need to create a `hollowverse-archive` org and move `commonpkg` to it so that it's not cluttering up the place here.

**Comment by forabi** [comment]
@msafih One thing is still unclear. What is `baseUrl` in `tsconfig.json` going to resolve to when the file is extended? Is it relative to the location of the extended file, which would be `./node_modules/common-config/` or is it relative to the location of the extending file, which would usually be the project's root. [The documentation](https://www.typescriptlang.org/docs/handbook/module-resolution.html#base-url) is not completely clear on this:

> value of baseUrl property in ‘tsconfig.json’ (if given path is relative, it is computed based on the location of ‘tsconfig.json’)

It does not mention which `tsconfig.json` is considered when resolving the value when extending the configuration. I'll have to test this before we merge.

**Comment by forabi** [comment]
I did a test on https://github.com/forabi/ts-baseurl-test/tree/extended-config-from-node-modules to see what the `baseUrl` value will be resolved to. It looks like it is resolved relative to `node_modules/common-config`. I updated the `tsconfig.json` in this PR to fix the value.

**Comment by msafih** [comment]
Cool. Should we also add a README.md to explain what this repo is for?

**Comment by forabi** [comment]
@msafih Added README file

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+     npm install https://github.com/hollowverse/common-config --save-dev
+     ```
+2. Install the [peer dependencies](./package.json#L7) for each of the tools used in the new project.
+3. Extend your project configuration files with the corresponding files from this package. For example, here is how to extend `tsconfig.json`:
+    
+    ```json
+    {
+      "extends": "./node_modules/common-config/tsconfig.json",
+      "compilerOptions": {
+        // ...
```
Comment (msafih) [inline README.md]: This looks like this when rendered

image

Comment (forabi) [inline README.md]: Removed

**Pull request merged**